packages/familie-header/src/icons/IkonSøk.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Ikon, { IkonProps } from './Ikon';
+
+const IkonSøk = ({ color = '#78706a', width = 16, height = 16 }: IkonProps) => {
```

Comment:
Kan man lagre denne fargen i en navngitt konstant slik at det er tydelig hvilken nav-farge det er?

Reply:
I will use nav-frontend colors instead🎨

Reply:
I dont know if the variables is available from tsx files, but you can try! 👍